KthLastElement.java

```diff
@@ -0,0 +1,46 @@
+package com.katermar;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Created by USER on 5/15/2017.
+ */
+public class KthLastElement {
+
+    private Integer value;

```

Comment:
You could use simple int here instead of Integer, unless you want to allow nulls being stored in the list.